server/rooms.ts

```diff
@@ -1368,6 +1368,12 @@ export class BasicChatRoom extends BasicRoom {
 		Rooms.rooms.delete(oldID);
 		Rooms.rooms.set(newID, this as ChatRoom);
 
+		for (const [alias, roomid] of Rooms.aliases.entries()) {
+			if (roomid === oldID) {
+				Rooms.aliases.set(alias, newID);
```

Comment:
You should delete the old aliases now that it's useless.

Reply:
It's an alias -> roomid map, I'm just updating it from `alias -> oldid` to `alias -> newid`

Reply:
Ah, right.